packages/server/routes/api/events.js

```diff
@@ -298,19 +298,21 @@ router.post(
     }
 
     try {
-      const { status, userId } = request.body;
-      const eventId = request.params.eventId;
+      const { userId, eventId } = matchedData(request);
       const event = await EventController.getEvent(eventId);
 
       const currentTime = new Date();
-      const checkInStartTime = new Date(event.startDate.getTime() - 5 * 60000);
-      const checkInEndTime = new Date(event.endDate.getTime() + 15 * 60000);
+
+      const EVENT_START_TIME = event.startDate.getTime() - 5 * 60000;
+      const EVENT_END_TIME = event.endDate.getTime() + 15 * 60000;
```

Comment:
```js
const EVENT_END_TIME = event.endDate.getTime() + 15 * 60000;
```
The above line is incorrect. How it is right now means 15 minutes after the event has ended. We want to stop allowing check in 15 minutes after the event has started so it should be
```js
const EVENT_END_TIME = event.startDate.getTime() + 15 * 60000;
```


The 60000 is still a magic number, so put it into a variable to clarify what exactly it's for which in this case is for unit conversion from minutes to milliseconds.
```js
const MINUTES_TO_MILLISECONDS = 60000;
```

- change `EVENT_START_TIME` to `eventCheckInStartBound`
- change `EVENT_END_TIME` to `eventCheckInEndBound`

These would be more specific and clearer names for these variables since they represent the beginning and end of the time window we're allowing people to check in. They don't represent the actual event start and end times.

```js
const MINUTES_TO_MILLISECONDS = 60000;
const eventCheckInStartBound = event.startDate.getTime() - 5 * MINUTES_TO_MILLISECONDS;
const eventCheckInEndBound = event.startDate.getTime() + 15 * MINUTES_TO_MILLISECONDS;
```

In the code block above, after getting rid of the magic numbers and using more specific variable names, it's more clear that we're allowing event check in 5 minutes before and ending check in 15 minutes after the start time.